cypress/support/fragments/settings/dataImport/fileExtensions/fileExtensions.js

```diff
@@ -107,4 +110,17 @@ export default {
       }),
     );
   },
+  creatNewFileExtension: (fileExtension) => {
```

Comment:
You don't need create a new function. You can use 
        FileExtensions.openNewFileExtensionForm();
        NewFileExtension.verifyNewFileExtensionFormIsOpened();
        NewFileExtension.fill(testData);
        NewFileExtension.save();

Reply:
i think it helpful and understandable , also for others 
but if you do not agree i can change

Reply:
User fill the data on the 'New file extension mapping' page. That's why I think that all related functions should be in  NewFileExtension fragment

Reply:
but Create is feature of File extension page for that create function i write in File Extension

Reply:
i mean user can create account from fileExtensions

Reply:
On the File extension page you only can click on the 'New' button

Reply:
what you think does it will be normal if i move creatNewFileExtension function in NewFileExtension fragment?

Reply:
yes, you can do this